test/consumption/2019-10-01/armconsumption/zz_generated_credits_client.go

```diff
@@ -36,7 +36,11 @@ func NewCreditsClient(credential azcore.TokenCredential, options *arm.ClientOpti
 	if len(cp.Host) == 0 {
 		cp.Host = arm.AzurePublicCloud
 	}
-	return &CreditsClient{ep: string(cp.Host), pl: armruntime.NewPipeline(module, version, credential, &cp)}
+	client := &CreditsClient{
+		host: string(cp.Host),
```

Comment:
Ugh, reading this has me thinking the `arm.ClientOptions` field shouldn't be "Host" because the value includes the scheme 😩

Reply:
Good point, it probably should have been `HostURL`.  Given the last round of pain from breaking changes, I don't think we want to fix it now.